.github/workflows/render-preview-deploy.yml

```diff
@@ -242,6 +242,16 @@ jobs:
               console.log(e)
             }
 
+      - name: Drop PostgreSQL PR database
```

Comment:
Also I think we should remove the label after deleting the db

Reply:
Since i have added it under the label destroy-review-app it will be removed automatically.